src/mobile/apps/artwork/templates/index.jade

```diff
@@ -7,7 +7,7 @@ block content
 
   include ../components/image/templates/index.jade
   include ../components/meta_data/templates/index.jade
-  if !user || !user.hasLabFeature('New Buy Now Flow')
+  if !sd.ENABLE_NEW_BUY_NOW_FLOW && !(user || user.hasLabFeature('New Buy Now Flow'))
```

Comment:
I think it should be `!(user && user.hasLabFeature('New Buy Now Flow'))` (`&&` instead of `||` in the lab feature check)